.arkana.yml

```diff
@@ -0,0 +1,12 @@
+result_path: 'dependencies' # Optional. Destination path of the generated code, related to the path of this config file. Defaults to ArkanaKeys.
```

Comment:
I suggestion a capital naming for this folder `Dependencies`.
I also think the default naming makes sense as is 🤔 
``` podfile
  pod "ArkanaKeys", path: "./.ArkanaKeys/ArkanaKeys"
```